schema/ext/vulnerability-1.0.xsd

```diff
@@ -0,0 +1,148 @@
+            <xs:element name="source" minOccurs="0" maxOccurs="1">
+                <xs:complexType>
+                    <xs:annotation>
+                        <xs:documentation>The source of the vulnerability</xs:documentation>
+                    </xs:annotation>
+                    <xs:sequence>
+                        <xs:element name="url" type="xs:string" minOccurs="0" maxOccurs="1">
+                            <xs:annotation>
+                                <xs:documentation>The url of the source</xs:documentation>
+                            </xs:annotation>
+                        </xs:element>
+                    </xs:sequence>
+                    <xs:attribute name="name" type="xs:normalizedString" use="required">
+                        <xs:annotation>
+                            <xs:documentation>The name of the source</xs:documentation>
+                        </xs:annotation>
+                    </xs:attribute>
+                </xs:complexType>
+            </xs:element>
+            <xs:element name="cvssV2" type="dg:cvssV2" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="cvssV3" type="dg:cvssV3" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="cwes" minOccurs="0" maxOccurs="1">
+                <xs:complexType>
+                    <xs:sequence minOccurs="0" maxOccurs="unbounded">
+                        <xs:element name="cwe" type="xs:string" minOccurs="0" maxOccurs="1"/>
+                    </xs:sequence>
+                </xs:complexType>
+            </xs:element>
+            <xs:element name="recommendations" minOccurs="0" maxOccurs="1">
+                <xs:complexType>
+                    <xs:annotation>
+                        <xs:documentation>The recommendations for the vulnerability</xs:documentation>
+                    </xs:annotation>
+                    <xs:sequence minOccurs="0" maxOccurs="unbounded">
+                        <xs:element name="recommendation" type="xs:string"/>
+                    </xs:sequence>
+                </xs:complexType>
+            </xs:element>
+            <xs:element name="description" minOccurs="0" maxOccurs="1">
+                <xs:complexType>
+                    <xs:annotation>
+                        <xs:documentation>The description of the vulnerability</xs:documentation>
+                    </xs:annotation>
+                    <xs:sequence>
+                        <xs:element name="explanation" type="xs:string" minOccurs="0" maxOccurs="1">
+                            <xs:annotation>
+                                <xs:documentation>An explanation of the vulnerability</xs:documentation>
+                            </xs:annotation>
+                        </xs:element>
+                        <xs:element name="detection" type="xs:string" minOccurs="0" maxOccurs="1">
+                            <xs:annotation>
+                                <xs:documentation>The detection of the vulnerability</xs:documentation>
```

Comment:
This needs more clarification. What is detection?

Reply:
> This needs more clarification. What is detection?

This is used as an attribute to describe vulnerabilities in sonatype products. Removed it considering the generic nature of this schema.  efa5e75